src/storyblok-auth-api/handle-requests/handleCallback/handleCallbackRequest.ts

```diff
@@ -54,34 +43,33 @@ export const handleCallbackRequest: HandleAuthRequest<{
     })
 
     if (!appSession) {
+      await adapter.removeCallbackData()
       return {
         type: 'error',
-        sessions: [clearCallbackData],
         redirectTo: params.errorCallback,
       }
     }
 
+    const spaceId = appSession.spaceId.toString()
+    const userId = appSession.userId.toString()
```

Comment:
can we include the appClientIdHere as well? In case there are two apps with the same space and user so that it is stored as a separate session? WDYT?